addons/mrp_byproduct/models/mrp_production.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from openerp import api, models, _
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    @api.multi
+    def action_confirm(self):
+        StockMove = self.env['stock.move']
```

Comment:
CASE:
If you take one variable suppose its occupy little bit 10kb space and suppose odoo have 10,000 users , That users install ***mrp_byproduct*** so for each users occupies 10kb memory
```
10,000user * 10kb = 1,00,000kb (~100mb)
```
so for this variable ~100mb occupies that really not needed and if you think because of performance , you don't worry because odoo server use distribution server , Its capable to manipulate this kind of thing.

Correct me if i am wrong because always performance is not issue some times memory is also issue.